assets/js/player.js

```diff
@@ -307,7 +311,7 @@ function updateCookie(newVolume, newSpeed) {
     var date = new Date();
     date.setFullYear(date.getFullYear() + 2);
 
-    var ipRegex = /^((\d+\.){3}\d+|[\dA-Fa-f]*:[\d:A-Fa-f]*:[\d:A-Fa-f]+)$/;
+    var ipRegex = /^(?:(?:\d+\.){3}\d+|[\dA-F]*:[\dA-F]*:[\d:A-F]+)$/i;
```

Comment:
Can you disable regex changes? Even if the new is more efficient, I find it less verbose.